lib/index.js

```diff
@@ -790,11 +796,11 @@ module.exports = () => {
        *  @summary Negate a private key.
        *  @return {Array} Negated private key.
        *  @throws {Error} Decode failed.
-       *  @arg {Array} key 32-byte private key to negate.
+       *  @argument {Array} key 32-byte private key to negate.
        */
-      function ecPrvkeyNegate(key) {
-        if (!key || !(key instanceof Uint8Array) || key.length !== 32) {
-          throw new TypeError('key must be a non-empty Uint8Array of 32 bytes');
+      function privateNegate(key) {
+        if (!key || !Buffer.isBuffer(key) || key.length !== 32) {
+          throw new TypeError('key must be a non-empty Buffer of 32 bytes');
```

Comment:
rever these changes and also change all `Buffer.isBuffer()` checks with `* instanceof Uint8Array`.